CHANGELOG.md

```diff
@@ -1,3 +1,159 @@
+# 1.2.0-rc4 (2020-03-1)
+## BREAKING CHANGE 
+This release uses new Identity Service (IS) artifact which is not backward compatible.
```

Comment:
@micahl - can you pls sign off on the wording of the doc here? Should we add more details on what this entails?

Reply:
I have confirmed w/ Micah on the terminology here :)

Reply:
@yophilav let's use the below to introduce the RC4...  And then I'll also go make sure the content in those links is in sync with the current state.

**AWARENESS**
This release contains a significant refactoring to the IoT Edge security daemon. It separates out the functionality for provisioning and providing cryptographic services for Linux-based devices into a set of stand-alone system services. Details on these individual system services can be found in the [Overview](https://azure.github.io/iot-identity-service) of the related github repository in which they reside. 

_Impact to Edge modules_
Every attempt has been made to ensure that the APIs on which Edge modules depend will remain unaffected and backward compatible. Issues affecting Edge modules will be treated with the highest priority. 

_Impact to installing / configuring IoT Edge_
The refactoring does affect the packaging, installation, and configuration of IoT Edge's host-level components. While we've attempted to minimize the impact of these, there are expected differences. For more details on these changes please refer to the discussion of [Packaging](https://azure.github.io/iot-identity-service/packaging.html).